CHANGELOG.md

```
@@ -14,6 +14,11 @@ and this project adheres to
 
 ## [0.0.16] - 2026-05-21
 
+### Added
+- ✨(frontend): display specific error when LLM provider is down
+
+## [0.0.16] - 2026-05-21
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)